autoprotocol/builders.py

```diff
@@ -1400,7 +1409,7 @@ def mode_params(self, liquid_class=None, position_x=None, position_y=None,
 
         Parameters
         ----------
-        liquid_class : Enum({"default", "air"}), optional
+        liquid_class : Enum({"default", "air", "viscous"}), optional
```

Comment:
I'd argue calling these an enum, when it is in fact a list, is misleading (enums are a python builtin as of 3.4 iirc). I think we should specify this is a list.

Reply:
Hmmm. liquid class is a list, but this parameter takes in a value from the list, instead of taking in the whole list as an argument, though?

Reply:
Actually, changed it to `str` since that's what the value type of liquid_class should be.